Surface_sweep_2/include/CGAL/No_intersection_surface_sweep_2.h

```diff
@@ -309,6 +309,78 @@ class No_intersection_surface_sweep_2 {
     m_visitor->after_sweep();
   }
 
+  /*!
+    Run the indexed sweep-line algorithm on a given range of
+    x-monotone curves and accessor. The main difference from the
+    original sweep() function is that the accessor allows to get
+    indices of end-points and avoid checking several times in the
+    event queue when the same vertex as several incident edges.
+
+    \param edges A range of edges
+    \param accessor An object providing, for a value_type `e` of
```

Comment:
You might want to introduce a concept since you are reusing it

Reply:
All of these documentations are internal (the sweep classes themselves are completely internal and not publicly documented). Do we have such things as internal concepts?

Reply:
OK I miss that point. No need then.